content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -211,7 +211,7 @@ so an earlier module has higher priority to allow or deny a request.
 
 ## Configuring the API Server using an Authorization Config File
 
-{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+{{< feature-state feature_gate_name="StructuredAuthorizationConfiguration" >}}
```

Comment:
Being late to the party ...
Can we simplify the parameter name from `feature_gate_name` to `name`?

Reply:
We could, but `feature_gate_name` is self documenting.

Reply:
The parameter `feature_gate_name` is not without a context. Within the given context, it can be simplified to `name` without introducing any ambiguity.

Reply:
> The parameter `feature_gate_name` is not without a context. Within the given context, it can be simplified to `name` without introducing any ambiguity.

You have to know that we have feature gates, and we shouldn't assume that our docs contributors are familiar with these.

We copy the feature _state_ from this other thing, _feature gates_. You and I know what these are @tengqm, but are we helping newcomers by shortening the parameter there? My opinion is that we are not.

Reply:
Sigh ... I mean, this parameter has context, every parameter in this shortcode is scoped to this shortcode. For example, in the existing shortcode `feature-state`, before this revision, we were using `state` rather than `feature_gate_state`. It is pretty obvious that the parameter is for.